.dependabot/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "daily"
+    ignored_updates:
+      - match:
+          dependency_name: "tap"
```

Comment:
We can go to latest `tap` now that we don't have to support older Node versions.

Reply:
Right, will remove it